packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -78,6 +79,36 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     return schemaComposer.buildSchema()
   }
 
+  /**
+   * Creates CRUD resolvers
+   *
+   * @param {GraphbackCoreMetadata} metadata - Core metatata containing all model information
+   */
+  public addResolvers(metadata: GraphbackCoreMetadata): IResolvers {
```

Comment:
The method name is misleading. When I read `addResolvers(metadata)`, it appears like we are adding resolvers to metadata but in fact what we are doing here is creating resolvers from metadata. Should this be renamed to `createResolvers` ?

Reply:
Good point. I will change that